src/dev/flang/ast/Feature.java

```diff
@@ -310,6 +310,15 @@ public Set<AbstractFeature> redefines()
   public boolean _addedLate = false;
 
 
+  /*
+   * true if this feature is found to be
+   * declared in a block with
+   * _newscope=true (e.g. if/else, loop)
+   * or in a case-block
+   */
```

Comment:
I do not fully understand this, imagine this code:
```
f0 =>
  if cc1 then
     f1 => 
       f2 =>
       if cc2 then
         f3 =>
     {
       f4 => 
     }
```
Then I assume  `f1` and `f3` are `_scoped` and `f0` is not. What about `f2` and `f4`?
Would be good to clarify this.